locations/spiders/talbots.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+
+class TemplateSpider(scrapy.Spider):
+    name = "talbots"
```

Comment:
```patch
+item_attributes = {"brand": "Talbots", "brand_wikidata": "Q7679064"}
```

Reply:
done. Thanks I didn't know how to get brand_wikidata.